README.md

```diff
@@ -136,10 +146,22 @@ Workmanager().registerOneOffTask("1", "simpleTask", initialDelay: Duration(secon
 
 > Not all constraints are mapped.
 
+- NetworkType
+  Constrains the type of network required for your work to run. For example, Connected.
+- RequiresBatteryNotLow
```

Comment:
For all of the below `requires*`, we need to check if they are 100% supported on iOS.

Reply:
Hey @ened 

I couldn't get the precise constraint on the iOS side, am not so familiar with the iOS side of things. From my mini research, this is what I got. Background task execution depends on several conditions, including:

1. User permission on “Background App Refresh”
2. Power supply from charger
3. Network connectivity

iOS fires an event to performFetchWithCompletionHandler when the network is in good condition and the device is not running in <a href="https://support.apple.com/en-hk/HT205234">Low-Power mode</a>.